visidata/features/errors_guide.py

```diff
@@ -0,0 +1,29 @@
+from visidata import GuideSheet, vd
+
+class ErrorsGuide(GuideSheet):
+    guide_text='''# A guide to errors in VisiData
+
+There are two severity levels VisiData will use when communicating something has gone wrong.
+If in performing an operation it reaches a [:166]warning[/], the operation will continue.
+With [:202]errors[/], operations will stop.
+
+[:202]Errors[/] and/or [:166]warnings[/] can manifest in different ways. How they appear determines how they are folllowed up on.
```

Comment:
`folllowed` is a typo.  How could we run these through a spell-checker?

Reply:
Seems I should run `pylint` locally (https://stackoverflow.com/questions/27162315/automated-docstring-and-comments-spell-check).

I don't think we want to add `pylint` to the CI. It became annoying for every project I was part of.

Reply:
totally, I don't like it either.  Maybe run it manually?  It might be a good step to include for doc authors.

Reply:
`pylint` did not catch 'folllowed' =/